### ISSUE TITLE
Problem with Yarn workspace

### ISSUE THREAD

**Issue by ehannes** [alampros/gatsby-plugin-resolve-src#12]
Hi!
This plugin looks great. I'm building a Gatsby theme, so I'm using [yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) according to the suggestion in the [documentation for building Gatsby themes](https://www.gatsbyjs.org/docs/themes/building-themes/).

Unfortunately, I cannot get this plugin to work with yarn workspaces. Here is a simple demo of a new gatsby site that I have added workspaces to: https://github.com/ehannes/gatsby-plugin-resolve-src-yarn-workspaces

I run `yarn workspace a gatsby develop` and I get this error:
```
 ERROR #98123  WEBPACK

 Generating development JavaScript bundle failed

 Can't resolve 'components/header' in '/project-path/gatsby-resolve-src/a/src/components'

 File: src/components/layout.js

 failed Building development bundle - 2.107s
```

**Comment by slorber** [comment]
+1

The following worked fine for me, put in gatsby-node:

```js
exports.onCreateWebpackConfig = ({ stage, actions, getConfig }) => {
  actions.setWebpackConfig({
    resolve: {
      modules: [
        // resolve src + yarn workspace node modules
        path.resolve(__dirname, 'src'),
        'node_modules',
        path.resolve(__dirname, 'node_modules'),
        path.resolve(__dirname, '../../node_modules'),
      ],
    },
  });
}
```

**Comment by alampros** [comment]
Sorry this took me so long! I've added a `srcPath` option to specify the path and published `v2.1.0`.
